apps/site/pages/components/organisms/cart-sidebar.mdx

```diff
@@ -0,0 +1,255 @@
+---
+description: The `CartSidebar` displays the summary of items added to the cart along with detailed information, and checkout option.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/CartSidebar___5863b0eb4a4a8ce9a98da5c760461e61.png
+---
+
+import path from 'path'
+import { useSSG } from 'nextra/ssg'
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import {
+  Alert,
+  CartSidebar,
+  CartSidebarList,
+  CartSidebarFooter,
+  UIProvider,
+  CartItem,
+  CartItemImage,
+  CartItemSummary,
+  Icon,
+  Gift,
+  GiftContent,
+  GiftImage,
+  OrderSummary,
+} from '@faststore/ui'
+
+import { product } from 'site/mocks/product'
+import { useFormattedPrice } from 'site/components/utilities/usePriceFormatter'
+
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const cartSidebarPath = path.resolve(__filename)
+  const components = ['CartSidebar.tsx']
+  const [CartSidebarProps] = getComponentPropsFrom(cartSidebarPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: {
+        CartSidebarProps,
+      },
+    },
+  }
+}
+
+export const CartSidebarPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { CartSidebarProps } = useSSG()
+  return {
+    CartSidebar: <PropsSection propsList={CartSidebarProps} />,
+  }[component]
+}
+
+export const skuActiveVariants =
+  product.isVariantOf.skuVariants.activeVariations
+export const activeVariations = Object.keys(skuActiveVariants).map((key) => ({
+  label: key,
+  option: skuActiveVariants[key],
+}))
+
+<header>
+
+# Cart Sidebar
+
+The `CartSidebar` displays the summary of items added to the cart along with detailed information, and checkout option. The final CartSidebar component is a compound of the following:
+
+</header>
+
+- `CartSidebarList`: wraps a series of `CartItem`s and `Gifts`.
+- `CartSidebarFooter`: wraps an `OrdeSummary` and probably the Checkout Button.
+
+## Import
+
+Import the component from [@faststore/ui](/../components)
+
+```tsx
+import { CartSidebar, CartSidebarList, CartSidebarFooter } from '@faststore/ui'
+```
+
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/organisms/CartSidebar/styles.scss'
+```
+
+---
+
+## Usage
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+    <OverviewSection containerStyle={{display: 'block'}}>
+      <UIProvider>
+      <CartSidebar totalItems={0} alertIcon={<Icon name="Truck" />} alertText="Free shipping starts at $300" >
```

Comment:
I got an error when I clicked on the close button from the `CartSidebar` using the preview.

Reply:
Can u try again? I've updated this doc many times haha.

The `CheckoutButton` was intentional.. do you think we should add it here?

Reply:
I'll try!
Maybe just a dumb button without any action?

Reply:
Okay! Lemme add it then :)